grakn-test/src/test/java/ai/grakn/test/graql/reasoner/Tests.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Grakn - A Distributed Semantic Database
+ * Copyright (C) 2016  Grakn Labs Limited
+ *
+ * Grakn is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Grakn is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Grakn. If not, see <http://www.gnu.org/licenses/gpl.txt>.
+ */
+
+package ai.grakn.test.graql.reasoner;
+
+import ai.grakn.graql.MatchQuery;
+import ai.grakn.graql.QueryBuilder;
+import ai.grakn.graql.internal.reasoner.query.QueryAnswers;
+import ai.grakn.test.GraphContext;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+
+import static ai.grakn.test.GraknTestEnv.usingTinker;
+import static org.junit.Assume.assumeTrue;
+
+public class Tests{
```

Comment:
`Tests` is a bad name for a test class I think. It provides no indication of what is being tested. What classes related to it etc. If I make a change and break this tests I won't have an idea where to start fixing it. 

Either rename or javadoc it.